### PR TITLE
[ET-1355] DQA Modifcations: PayPal Section

### PR DESCRIPTION
**Pull request by codingmusician**
### 🎫 Ticket

[ET-1355] 

### 🗒️ Description

- Added space above submit button
- Redirect to PayPal section after connecting to PayPal

### 🎥 Artifacts 

DQA: https://www.figma.com/file/TvIXoUEJAGb0AptZI260UO/TicketsCommerce?node-id=1732%3A307

### ✔️ Checklist
- [ ] I've included a changelog entry in the readme.txt file. 
- [x] My code is tested. 
- [x] My code has [proper inline documentation](https://the-events-calendar.github.io/products-engineering/docs/code-standards/).

[ET-1355]: https://theeventscalendar.atlassian.net/browse/ET-1355?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/Tickets/Commerce/Gateways/PayPal/REST/On_Boarding_Endpoint.php [hunk 1]
```diff
@@ -122,7 +123,7 @@ public function handle_signup_redirect( WP_REST_Request $request ) {
 		$signup        = tribe( Signup::class );
 		$existing_hash = $signup->get_transient_hash();
 		$request_hash  = $request->get_param( 'hash' );
-		$return_url    = Tribe__Settings::instance()->get_url( [ 'tab' => 'payments' ] );
+		$return_url    = Tribe__Settings::instance()->get_url( [ 'tab' => 'payments', 'tc-section' => tribe( Gateway::class )->get_key() ] );
```
Comment (juanfra) [inline src/Tickets/Commerce/Gateways/PayPal/REST/On_Boarding_Endpoint.php:126]: We can maybe get the key for the payments tab from `Payments_Tab`

File: src/Tickets/Commerce/Gateways/PayPal/REST/On_Boarding_Endpoint.php [hunk 2]
```diff
@@ -122,7 +124,10 @@ public function handle_signup_redirect( WP_REST_Request $request ) {
 		$signup        = tribe( Signup::class );
 		$existing_hash = $signup->get_transient_hash();
 		$request_hash  = $request->get_param( 'hash' );
-		$return_url    = Tribe__Settings::instance()->get_url( [ 'tab' => 'payments' ] );
+		$return_url    = Tribe__Settings::instance()->get_url( [
+			'tab' => 'payments',
```
Comment (juanfra) [inline src/Tickets/Commerce/Gateways/PayPal/REST/On_Boarding_Endpoint.php:128]:
```suggestion
			'tab' => Payments_Tab::$slug,
```

File: src/admin-views/settings/tickets-commerce/paypal/connect/active/actions/refresh-connection.php [hunk 3]
```diff
@@ -6,19 +6,26 @@
...
 
+use TEC\Tickets\Commerce\Payments_Tab;
+
 if ( empty( $is_merchant_connected ) ) {
 	return;
 }
 
-$resync_connection_url = Tribe__Settings::instance()->get_url( [ 'tab' => 'payments', 'tc-action' => 'paypal-resync-connection' ] );
+$resync_connection_url = Tribe__Settings::instance()->get_url( [ 
+	'tab' => 'payments',
```
Comment (juanfra) [inline src/admin-views/settings/tickets-commerce/paypal/connect/active/actions/refresh-connection.php:25]:
```suggestion
	'tab' => Payments_Tab::$slug,
```

File: src/admin-views/settings/tickets-commerce/paypal/connect/active/actions/refresh-user-info.php [hunk 4]
```diff
@@ -12,15 +12,22 @@
...
+
 $debug = defined( 'WP_DEBUG' ) && WP_DEBUG;
 
 if ( empty( $is_merchant_connected ) || ! $debug ) {
 	return;
 }
 
-$url = Tribe__Settings::instance()->get_url( [ 'tab' => 'payments', 'tc-action' => 'paypal-refresh-user-info' ] );
+$url = Tribe__Settings::instance()->get_url( [
+	'tab' => 'payments',
```
Comment (juanfra) [inline src/admin-views/settings/tickets-commerce/paypal/connect/active/actions/refresh-user-info.php:27]:
```suggestion
	'tab' => Payments_Tab::$slug,
```

File: src/admin-views/settings/tickets-commerce/paypal/connect/active/actions/refresh-webhook.php [hunk 5]
```diff
@@ -12,15 +12,22 @@
...
+
 $debug = defined( 'WP_DEBUG' ) && WP_DEBUG;
 
 if ( empty( $is_merchant_connected ) || ! $debug ) {
 	return;
 }
 
-$url = Tribe__Settings::instance()->get_url( [ 'tab' => 'payments', 'tc-action' => 'paypal-refresh-webhook' ] );
+$url = Tribe__Settings::instance()->get_url( [
+	'tab' => 'payments',
```
Comment (juanfra) [inline src/admin-views/settings/tickets-commerce/paypal/connect/active/actions/refresh-webhook.php:27]:
```suggestion
	'tab' => Payments_Tab::$slug,
```

**Review by juanfra — CHANGES_REQUESTED** [review]
For all the existing templates that we're updating, we should set the `@version TBD` and then add a `@since TBD Explanation of the changes`.

**Review by juanfra — APPROVED** [review]
🚀

**Comment by codingmusician** [comment]
There was a test that needed the new template variable added into it and an updated snapshot.

**Pull request merged**